lib/core.js

```diff
@@ -114,6 +115,13 @@ module.exports = class Core {
     if (this.exclusive !== null) this.exclusive.unlock()
   }
 
+  _ensureEncryptionContext () {
+    if (this.encryptionContext) return
+    this.encryptionContext = {}
```

Comment:
lets ensure the shape here immediately, like so

```
this.encryptionContext = {
  key: this.key,
  manifest: this.manifest
}
```

both are inited to null here so should be fine